src/ffi.rs

```diff
@@ -234,6 +234,36 @@ pub extern fn quiche_config_set_cc_algorithm(
     config.set_cc_algorithm(algo);
 }
 
+#[no_mangle]
+#[cfg(feature = "qlog")]
+pub extern fn quiche_conn_set_qlog(
```

Comment:
We should call this something like `quiche_conn_set_qlog_path` to be more specific.
```suggestion
pub extern fn quiche_conn_set_qlog_path(
```